### ISSUE TITLE
What dataset is used to train the model from the custom dataset?

### ISSUE THREAD

**Issue by FadhilRa** [hsiangwei0903/Deep-EIoU#23]
Hi, I'm curious about your project.
In this project there is session "Demo on custom dataset". In the first step we have to prepare the model for Demo on custom dataset. You give us detector model (YOLOX Detector) and OSNet ReID Model. I'm curious what dataset is use to train this model? Especially on YOLOX Detector. Can you explain about this? because I'm still learning about Occlusion in Sport and want to learn more about this project

Thank you!!

**Comment by hsiangwei0903** [comment]
YOLO_X and OSNet ReID model used in demo.py are both trained on the SportsMOT dataset, following the training setting in our paper.

**Comment by FadhilRa** [comment]
Hi, thank you for your response!

I'm interested in understanding more about the training setup. Could you clarify which version of YOLOX was used to train the detector model on the SportsMOT dataset? Did you use any specific modifications or configurations during training?

Looking forward to your response. Thanks!

**Comment by hsiangwei0903** [comment]
We run the yolox training using [ByteTrack](https://github.com/ifzhang/ByteTrack) code base with the exact same config.

**Comment by FadhilRa** [comment]
Thank you for your response!

Just to clarify, was ByteTrack used only for training the YOLOX detector, or is ByteTrack still being used in this project for tracking as well?

**Comment by hsiangwei0903** [comment]
We use ByteTrack's codebase to train our YOLOX detector, our tracking algorithm DeepEIoU is different from bytetrack.

**Comment by hsiangwei0903** [comment]
closed the issue for now, feel free to reopen if any more question.

**Comment by FadhilRa** [comment]
Thank you for your responses so far!

I would like to ask for more details regarding the training setup for the YOLOX detector on the SportsMOT dataset. Specifically:
1. What hardware specifications were used during training (e.g., GPU model, number of GPUs, RAM, etc.)?
2. How long did the training process take?
3. Did you use any specific hyperparameter settings (batch size, learning rate, epochs, etc.), or did you follow the default ByteTrack settings?
4. Were there any additional modifications or augmentations applied to the dataset during training?
I appreciate your time in answering these questions. Looking forward to your response!

**Comment by hsiangwei0903** [comment]
1. we use 8 V100 to train the YOLOX model
2. should be less than one day, depends on how many data you used
3. every hyperparameter follow ByteTrack's YOLOX training setting
4. we use the same training setting and data augmentations as ByteTrack's YOLOX training

**Comment by FadhilRa** [comment]
Thank you for the information!

May I ask how much VRAM each of the V100 GPUs had during training? I’d like to check whether my current hardware setup would be sufficient to replicate the experiment.

Thank you in advance!
